Compilation error: ‘test_exit_grid’ not specified in enclosing ‘parallel’

Another compilation error, this time related to openmp:

```
gcc -static-libstdc++ -fopenmp -c -DPOINTER_SIZE=8 optical_depth.f90 -o optical_depth.o 
optical_depth.f90:356:0:

            if (test_exit_grid(icell0, x0, y0, z0)) exit inf_loop
 
Error: ‘test_exit_grid’ not specified in enclosing ‘parallel’
optical_depth.f90:319:0:

      !$omp private(icell,previous_cell,next_cell,icell0,x0,y0,z0,x1,y1,z1,norme,u,v,w,l,l_contrib,l_void_before,ltest,factor,sum)
 
Error: enclosing ‘parallel’
make: *** [Makefile:195: optical_depth.o] Error 1

```

--- a/src/optical_depth.f90
+++ b/src/optical_depth.f90
@@ -316,7 +316,7 @@ subroutine compute_column(type, column, lambda)
   do direction = 1, n_directions
      !$omp parallel default(none) &
      !$omp shared(densite_gaz,lVoronoi,Voronoi,direction,column,r_grid,z_grid,phi_grid,n_cells,cross_cell,CD_units,kappa,lambda,type) &
-     !$omp private(icell,previous_cell,next_cell,icell0,x0,y0,z0,x1,y1,z1,norme,u,v,w,l,l_contrib,l_void_before,ltest,factor,sum)
+     !$omp private(icell,previous_cell,next_cell,icell0,x0,y0,z0,x1,y1,z1,norme,u,v,w,l,l_contrib,l_void_before,ltest,factor,sum,test_exit_grid)
      !$omp do
      do icell=1,n_cells
         if (lVoronoi) then

